### DISCUSSION TITLE
Range, Fuel Consumption, and External Temp

### DISCUSSION THREAD

**Discussion by DanGarion** [G-Two/homeassistant-subaru#26] [General]
Did something happen with Range, Fuel Consumption, and External Temp? I had these working before but now the sensors/entities aren't matching up correctly. What changed?

**Comment by G-Two** [comment]
Everything has been working for me over the past two weeks. Are you still having issues?

**Comment by DanGarion** [comment]
Yeah it's working again. I wonder what was going on when I posted this.
